apps/explorer/src/pages/epochs/EpochDetail.tsx

```diff
@@ -0,0 +1,164 @@
+                        <SuiAmount amount={gasCostSummary?.storageRevenue} />
+                    </Stats>
+                    <Stats label="Stake Rewards" tooltip="Stake Rewards">
+                        <SuiAmount amount={gasCostSummary?.stakeRewards} />
+                    </Stats>
+                </EpochStats>
+                <EpochStats label="Rewards">
+                    <Stats label="Stake Subsidies" tooltip="Stake Subsidies">
+                        <SuiAmount amount={stakeSubsidies} />
+                    </Stats>
+                    <Stats label="Total Rewards" tooltip="Total Rewards">
+                        <SuiAmount amount={totalRewards} />
+                    </Stats>
+
+                    <Stats
+                        label="Storage Fund Earnings"
+                        tooltip="Storage Fund Earnings"
+                    >
+                        <SuiAmount amount={storageFundEarnings} />
+                    </Stats>
+                </EpochStats>
+                <Card spacing="lg">
+                    <RingChart
+                        title="Validators in Next Epoch"
+                        suffix="validators"
+                        data={[
+                            {
+                                value: active ?? 0,
+                                label: 'Active',
+                                color: '#589AEA',
+                            },
+                            {
+                                value: pending ?? 0,
+                                label: 'New',
+                                color: '#6FBCF0',
+                            },
+                            {
+                                value: atRisk ?? 0,
+                                label: 'At Risk',
+                                color: '#FF794B',
+                            },
+                        ]}
+                    />
+                </Card>
+            </div>
+
+            <TabGroup size="lg">
+                <TabList>
+                    <Tab>Checkpoints</Tab>
+                    <Tab>Participating Validators</Tab>
+                </TabList>
+                <TabPanels className="mt-4">
+                    <TabPanel>
+                        <CheckpointsTable epoch={epochQuery.data.epoch} />
+                    </TabPanel>
+                    <TabPanel>
+                        {validatorsTable ? (
+                            <TableCard
+                                data={validatorsTable?.data!}
+                                sortTable
+                                defaultSorting={[{ id: 'stake', desc: false }]}
+                                columns={validatorsTable?.columns!}
+                            />
+                        ) : null}
+                    </TabPanel>
+                </TabPanels>
+            </TabGroup>
+        </div>
+    );
+}
+
+export default function EpochDetailFeatureFlagged() {
+    const gb = useGrowthBook();
+    if (gb?.ready) {
```

Comment:
set in growthbook - they also export a `FeaturesReady` helper, but that doesn't work. I was just trying to avoid writing anything more complex, but if we're going to want to feature gate routes in the future, it might be worth it.